modcc/parser.cpp

```diff
@@ -399,9 +399,17 @@ void Parser::parse_state_block() {
         }
 
         parm.token = token_;
-        //state_block.state_variables.push_back(token_.spelling);
         get_token();
 
+        if(token_.type == tok::from) {
+            // silently skips from/to
+            from_to_description();
+            if (status_ == lexerStatus::error) {
+                error(pprintf("range description must be of form FROM <int> TO <int>: '%'", token_));
```

Comment:
I don't think you need to set the error message here: it was already set in the call to `from_to_description()`, right?